pkg/networkmanager/firewall-client.js

```diff
@@ -335,6 +335,22 @@ cockpit.spawn(['sh', '-c', 'pkcheck --action-id org.fedoraproject.FirewallD1.all
             firewall.readonly = false;
             firewall.debouncedEvent('changed');
             firewall.debouncedGetZones();
+        })
+        .catch(error => {
+            console.warn("pkcheck failed", error);
```

Comment:
This is too broad -- the command also fails if `pkcheck` is installed, but the user does not have the permission. This needs to inspect the error and only fall back on "command not found". Also, this should not be a `warn`, I think.

Reply:
That's actually a question how it should behave. It depends on priorities. You do not have the PolicyKit permission but you are admin (`root`). What should have a higher priority?

I was already thinking about this and decided to use the same analogy as with the file system permissions. If you remove all permissions from a file (`chmod a-rw`) you can still read (`cat`) or write (`echo >>`) to the file if you are `root`.

On the other hand I'm not a PolicyKit expert and I do not know how this problem is solved elsewhere, it should be consistent...

Reply:
root has all the polkit permissions anyway, at least for default rules that are shipped by projects. I was mostly worried about doing (essentially) the same check twice, as this further slows down the page for non-admins. But you are right about the consistency argument. OK, then let's please just tone it down to console.log(), and see how the PR turns out otherwise. Thanks!